vere/http.c

```diff
@@ -2289,14 +2299,31 @@ _proxy_ward_timer_cb(uv_timer_t* tim_u)
 static void
 _proxy_ward_plan(u3_ward* rev_u)
 {
+  u3_noun non;
+
+  {
+    c3_w* non_w = c3_malloc(64);
+    c3_w  len_w;
+
+    c3_rand(non_w);
+
+    non = u3i_words(16, non_w);
+    len_w = u3r_met(3, non);
+
+    //  the nonce is saved to authenticate u3_wcon
+    //  and will be freed with u3_ward
+    //
+    rev_u->non_u = uv_buf_init((c3_c*)non_w, len_w);
```

Comment:
The earlier call to uv_buf_init() used len_w + 1. Are you sure this is the right number here?

I had a physics professor who, when called out on a blackboard mistake, said that factors of 2 will plague you for the rest of your life. Off-by-1 errors seem like the CS equivalent.

Reply:
To handle leading zeros, the nonce is length-prefixed when sent from the planet back to the star of tcp -- hence the +1. The length is confirmed and then stripped when the nonce is validated by the star in `_proxy_wcon_peek_read_cb()`. Perhaps not the clearest implementation, but it's correct.